libs/standard-tests/langchain_standard_tests/integration_tests/chat_models.py

```diff
@@ -170,7 +170,14 @@ def test_stop_sequence(self, model: BaseChatModel) -> None:
     def test_tool_calling(self, model: BaseChatModel) -> None:
         if not self.has_tool_calling:
             pytest.skip("Test requires tool calling.")
-        model_with_tools = model.bind_tools([magic_function])
+        if self.tool_choice_value == "dict":
```

Comment:
this format is openai-specific (eg anthropic doesn't support i dont think). i actually tihnk more models support just passing in tool name as a string to bind_tools, and think thats what we'd want to standardize on from a devx perspective anyways

Reply:
Thanks, updated this to tool name.